packages/tokens/package.json

```diff
@@ -15,10 +15,17 @@
   },
   "license": "MPL-2.0",
   "author": "HashiCorp Design Systems <design-systems@hashicorp.com>",
+  "files": [
```

Comment:
in the `flight` package we're using a `.npmignore` file (see https://github.com/hashicorp/design-system/blob/main/packages/flight-icons/.npmignore) while here I see we use the [files](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files) field. maybe we should use a standard approach? (and in that case, I vote for `.npmignore` that is more explicit).

Reply:
I pushed a commit that follows this suggestion which feels ok for me for now. I think I might argue that using `files` feels cleaner as it seems cleaner to opt in to the things you know you want vs having to itemize the list of things you don't want (which is a lot in both cases). No strong opinions here as I don't have a ton of experience with this 🤷

Reply:
Both solutions work for me too. My point was about choosing one of the two strategies and use the same across the entire monorepo.

Reply:
Makes sense. `.npmignore` is also what ember projects use by default.